handout/directives/ng_style_directive.md

```diff
@@ -24,3 +24,27 @@ export class StyleExampleComponent {
 [View Example](https://plnkr.co/edit/Vcpt1vDWryaiYcmoQEyC?p=preview)
 
 Notice that binding a directive works the exact same way as component attribute bindings. Here, we're binding an expression, an object literal, to the `ngStyle` directive so the directive name must be enclosed in square brackets. `ngStyle` accepts an object whose properties and values define that element's style. In this case, we can see that both kebab case and lower camel case can be used when specifying a style property. Also notice that both the html `style` attribute and Angular 2 `ngStyle` directive are combined when styling the element.
+
+We can remove the style properties out of the template into the component as a property object, which then gets assigned to `NgStyle` using property binding. This allows dynamic changes to the values as well as provides the flexibility to add and remove style properties. One thing to be aware of, the style property names must be in quotes.
+
+```typescript
+@Component({
+  selector: 'style-example',
+  template: `
+    <p style="padding: 1rem"
+        [ngStyle]="alertStyles">
+        <ng-content></ng-content>
+    </p>
+  `
+})
+export class StyleExampleComponent {
+  borderStyle: string = '1px solid black';
+
+  // Must be JSON valid.
```

Comment:
I'd remove this comment, it's a plain object. The property values can also be functions that return values.